docs/make.jl

```diff
@@ -29,7 +29,8 @@ end
 
 # Auxiliar function that returns an array with the subseccions of a literate seccion
 function pages_for_literate_seccion(folder)
-    return [joinpath("generated", folder, f) for f in readdir(joinpath(gen, folder)) if is_md_file(f)]
+    literate_examples = [joinpath("generated", folder, f) for f in readdir(joinpath(gen, folder)) if is_md_file(f)]
+    return push!(literate_examples, "pluto-simulation.md")
```

Comment:
A more elegant solution?

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/248/commits/7e191c1ee9c0decfe6be83102438be868ad88e8a

Reply:
Create issue to change the way the literate examples are being generated

Reply:
Issue #251